seed/serializers/pint.py

```diff
@@ -78,13 +78,14 @@ def pretty_units_from_spec(unit_spec):
     return pretty_units(quantity)
 
 
-def add_pint_unit_suffix(organization, column):
+def add_pint_unit_suffix(organization, column, data_key="data_type", display_key="display_name"):
     """
     transforms the displayName coming from `Column.retrieve_all` to add known
     units where applicable,  eg. 'Gross Floor Area' to 'Gross Floor Area (sq.
     ft.)', using the organization's unit preferences.
     """
 
+    # add units to the end of the field
```

Comment:
Sorry to be picky - think we could remove the comments on lines 88, 97, 103, and 114? As much as we can, we've tried to lean towards descriptive method and variable names if possible in this codebase (i.e. if you thought `format_column_name` wasn't descriptive enough, I'd be cool if you changed it since it's an inner function - maybe `append_display_units` 🤷 .) Ideally, we'd reserve in-line comments for odd or one-off clarifications.

Reply:
Sorry!  I've always been in the habbit of adding comments to all of my code, rather than simply to be more descriptive.